### PR TITLE
Replace postman api documentation

### LINKED ISSUE #100: Look for a Postman alternative
**Description:**

### PR DESCRIPTION
**Pull request by TomRomeo**
## Description
This PR replaces the postman api documentation with a documentation in the format of [Bruno](https://github.com/usebruno/bruno), an open source IDE for exploring APIs.

Some benefits of this solution include:
- every request is a separate file, making additions and changes more easily comprehensible in upcoming pull requests
- cli client for testing available
- requests can be extended with pre- and post-response scripting, assertions and tests

Currently, a few scripts to simplify api testing have already been added, for example:
- after logging in with the login route, the access_token is automatically added to every following request
- after creating a group/deck/card, the new ID is added to the environment to easily follow up with other requests

resolves #100

### CODE REVIEW

**Review by darmiel — APPROVED** [review]
tobi approve

**Review by Julian702 — APPROVED** [review]

File: backend/api-documentation/cards/delete.bru [hunk 1]
```diff
@@ -0,0 +1,15 @@
+meta {
+  name: delete
```
Comment (4KevR) [inline backend/api-documentation/cards/delete.bru:2]: The naming for the delete and modify is inconsistent in some requests. Sometimes the entity is written out, as in `delete user` but for specific objects like card it is just `delete`. Maybe we want to write it out in every request.

**Review by 4KevR — APPROVED** [review]
Looks great! I have just one comment up for discussion but it's not quite relevant.

File: backend/api-documentation/cards/delete.bru (hunk 1 above)
Comment (TomRomeo) [reply backend/api-documentation/cards/delete.bru]: Yes, that sounds like a good idea, I renamed the requests for uniformity in 272aad5

**Pull request merged**